LOs.csv

```diff
@@ -0,0 +1,136 @@
+LO-3.2.2.1-2,(K2),Understand how multiple lines can be used to define scalar variables,,,
+LO-3.2.2.2,(K2),Understand how to access primitive data types,,,
+LO-3.2.2.3,(K2),Understand how to set and access data in list variables,,,
+LO-3.2.2.4,(K2),Understand how to set and access data in dict variables,,,
+LO-3.2.3,(K3),Be able to assign return values from keywords to variables,,,
+LO-3.2.4,(K2),Understand how to create variables using the VAR statement,,,
+LO-3.2.5,(K2),Understand how `local` and `suite` scope variables are created,,,
+LO-3.3.2,(K1),Recall the rules how keyword names are matched.,,,
+LO-3.3.3,(K1),Recall all available settings and their purpose for User Keywords,,,
+LO-3.3.4,(K1),Recall the significance of the first logical line and in keyword documentation for the log file.,,,
+LO-3.3.5,(K2),Understand the purpose and syntax of the [Arguments] setting in User Keywords.,,,
+LO-3.3.5.1-1,(K1),Recall what makes an argument mandatory in a user keyword.,,,
+LO-3.3.5.1-2,(K3),Define User Keywords with mandatory arguments.,,,
+LO-3.3.5.2-1,(K1),Recall how to define optional arguments in a user keyword.,,,
+LO-3.3.5.2-2,(K3),Define User Keywords with optional arguments.,,,
+LO-3.3.5.3-1,(K2),Describe how embedded arguments are replaced by actual values during keyword execution.,,,
+LO-3.3.5.3-2,(K2),Understand the role of embedded arguments in Behavior-Driven Development (BDD) style.,,,
+LO-3.3.6-1,(K2),Understand how the `RETURN` statement passes data between different keywords.,,,
+LO-3.3.6-2,(K3),Use the `RETURN` statement to return values from a user keyword and assign it to a variable.,,,
+LO-3.3.7,(K1),Recall the naming conventions for user keywords.,,,
+LO-3.4,(K2),Understand the basic concept and syntax of Data-Driven Specification,,,
+LO-3.4.1-1,(K2),Understand how to define and use test|task templates,,,
+LO-3.4.1-2,(K1),Recall the differences between the two different approaches to define Data-Driven Specification,,,
+LO-3.4.1.1,(K1),Recall the syntax and properties of multiple named test|task with one template,,,
+LO-3.4.1.2,(K1),Recall the syntax and properties of named test|task with multiple data rows,,,
+LO-3.5,(K1),Recall that naming conflicts can arise from the import of multiple resource files.,,,
+LO-3.5.1,(K2),Understand how transitive imports of resource files and libraries work.,,,
+LO-3.5.2,(K3),Be able to configure a library import using arguments.,,,
+LO-3.5.3,(K2),Explain how naming conflicts can happen and how to mitigate them.,,,
+LO-4.1-1,(K1),Recall the purpose and benefits of Setups in Robot Framework,,,
+LO-4.1-2,(K1),Recall the different levels where a Setup can be defined,,,
+LO-4.1.1-1,(K1),"Recall key characteristics, benefits, and syntax of Suite Setup",,,
+LO-4.1.1-2,(K2),Understand when Suite Setup is executed and used,,,
+LO-4.1.2-1,(K1),"Recall key characteristics, benefits, and syntax of Test Setup",,,
+LO-4.1.2-2,(K2),Understand when Test|Task Setup is executed and used,,,

```

Comment:
This `Test|Task`-syntax was broken in the render version, removing the part after `|`.